README.md

```diff
@@ -179,6 +179,8 @@
 
 [Joyent](https://www.joyent.com/blog/manta-postmortem-7-27-2015). Operations on Manta were blocked because a lock couldn't be obtained on their PostgreSQL metadata servers. This was due to a combination of PostgreSQL's transaction wraparound maintenance taking a lock on something, and a Joyent query that unnecessarily tried to take a global lock.
 
+[Joyent](https://www.joyent.com/blog/postmortem-for-outage-of-us-east-1-may-27-2014) see also [Bryan Cantrill's talk](https://www.youtube.com/watch?v=30jNsCVLpAE) ([slides](https://www.slideshare.net/bcantrill/debugging-under-fire-keeping-your-head-when-systems-have-lost-their-mind)). An operator used a tool with lax input validation to reboot a small number of servers undergoing maintenance, but forgot to type `-n` and instead rebooted all servers in the datacenter. This caused an outage that lasted 2.5 hours, rebooted all customer instances, put trememndous load on DHCP/TFTP PXE boot systems, and left API systems (by design) requiring manual intervention. The `sdc-oneachnode` tool was subsequently [improved](https://github.com/joyent/sdc-sdc/commit/f9b7525eded5d1e5c6f609f3849c2331bbf54562) with better input validation.
```

Comment:
```suggestion
[Joyent](https://www.joyent.com/blog/postmortem-for-outage-of-us-east-1-may-27-2014). An operator used a tool with lax input validation to reboot a small number of servers undergoing maintenance, but forgot to type `-n` and instead rebooted all servers in the datacenter. This caused an outage that lasted 2.5 hours, rebooted all customer instances, put tremendous load on DHCP/TFTP PXE boot systems, and left API systems requiring manual intervention. See also [Bryan Cantrill's talk](https://www.youtube.com/watch?v=30jNsCVLpAE).
```

Reply:
I simplified this a bit, also kept the formatting to match the rest of the repo.

Reply:
Thanks! I appreciate the help with conciseness. Picking which bits of my research to let go of is always difficult.

I've applied your changes as Co-authored-by, removed a comma, and pushed to this branch.